h2o-scala/src/test/scala/water/TestBase.scala

```diff
@@ -0,0 +1,16 @@
+package water
+
+import org.scalatest.FunSuite
+import water.fvec.Vec
+
+/**
+  * Created by vpatryshev on 11/12/16.
+  */
+class TestBase extends FunSuite {
+  {
+    // need this because -ea IS NOT ALWAYS set in intellij
+    getClass().getClassLoader().setDefaultAssertionStatus(true)
```

Comment:
It is dangerous, because it setups EA only on a selected class loader. It will not trigger the same setup of all used class loaders (for example, there is javassist class loader).